openapi/openapi.yaml

```diff
@@ -697,6 +697,50 @@ components:
       pattern: ^((([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)$
       example: '1.0.1'
 
+    Tag:
+      description: 'Tag'
+      title: 'Tag'
+      type: object
+      properties:
+        name:
+          maxLength: 32
+          type: string
+        id:
```

Comment:
Tags are not referenced by `id` anywhere.

Reply:
@newswangerd @chouseknecht How close does the new /_ui/tags need to resemble the v2 schema as mentioned in "Add tags schemas and endpoints for _ui. The schema will match the v2 schema." (https://github.com/ansible/galaxy-dev/issues/58#issue-474134715)

Reply:
I don't see the need for the db specific and mutable stuff like 'id', 'modified', 'created', etc.
Not sure what @newswangerd needs/users in the ui though, though id/modified/created for tags seems unlikely to be useful